samples/bot-formatting-cards/csharp/README.md

```diff
@@ -125,6 +125,46 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
             }
             ]
         ```
+   
+   **Note: The users are in the card component have to be on the same tenant.**
```

Comment:
Mention properly that, In Adaptive Card what we are defining (User details) should be exist in the same tenant where you are testing the app (teams login) etc...

Reply:
Fixed